### DISCUSSION TITLE
Repository information a user being shown in another user profile

### DISCUSSION THREAD

**Discussion by iamrafaelmelo** [community/community#18193] [Other Feature Feedback, Questions, & Ideas] [answered]
I was browsing some repositories when I came across the following situation, remembering that this repository was the last one I saw before entering this user's profile.

![Screenshot 2022-06-09 195029](https://user-images.githubusercontent.com/52138516/172958688-5feb6b20-99a2-45b1-a9ff-361c0de63def.png)

**Comment by ghostofcodes** [comment, marked as answer]
The user probably achieved or private the repository? That's more likely the most probable reason

**Reply by iamrafaelmelo** [reply]
right, but what would be the reason for this information being displayed on another user's profile?

**Reply by ghostofcodes** [reply]
That's pretty odd yeah, it might be due to a glitch? Have you tried refreshing the page and the same error is still popping up?
